packages/babel-preset-default/index.js

```diff
@@ -8,7 +8,6 @@ module.exports = function( api ) {
 				targets: {
 					browsers: [ 'extends @wordpress/browserslist-config' ],
 				},
-				useBuiltIns: 'usage',
 			} ],
 			isTestEnv && [ '@babel/preset-env', {
 				useBuiltIns: 'usage',
```

Comment:
I had to revert the change which removes:
`useBuiltIns: 'usage',`
from test environment because some tests were failing. Those which use `Array.values` method which isn't supported by Node.

@youknowriad suggested using `babel-polyfill` instead to have the setup closer to the browser but I found out that it can cause memory issues with Jest. See https://github.com/facebook/jest/pull/2755 and the line in the changelog:
> Breaking Change: Replaced auto-loading of babel-polyfill with only regenerator-runtime, fixes a major memory leak.